lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AbstractCommandBuilder.cs

```diff
@@ -16,22 +16,56 @@
 // under the License.
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
+using Org.Apache.REEF.Common.Files;
 
 namespace Org.Apache.REEF.Client.AzureBatch.Util
 {
     internal abstract class AbstractCommandBuilder : ICommandBuilder
     {
-        public string BuildDriverCommand()
+        private const string JAVA_EXE = @"java";
```

Comment:
I believe that in C# the standard way to name constants is camel-case with first letter uppercase, ie. JavaExe. I took a quick look at how these variables are named elsewhere and REEF.NET seems to be following that convention.

Reply:
Fixed. But REEFFileNames.cs is using this format.